### PR TITLE
1426 single classification chi square test as test against total

### PR DESCRIPTION
**Pull request by alexbuchhammer**
This PR introduces a new `Test` method, `chi_square_sc(level=0.05, return_diagnostics=False)`. It is required to produce the Quantum-like tests against total, that follow a completely different approach than Dimensions (and Askia).

I have modified the respective code in the main test algorithm to combine the new Chi^2-test results with the established output computation. I am doing this by skipping the "regular" test against total results and instead append the Chi^2 ones.

Some notes:
* I have also modified the `view_maps` to always force the total test for Link, i.e. it's enough to instruct the tests as per 
`b.set_sigtests(levels=0.05, mimic='LINK_legacy')` in **NB5**; NB6 and NB8 can be run as usual. 
* Test results are indicated as before using the green-up & red-down arrows.
* Chi^2 tests are not performed on means.

Documentation can be found in our Confluence pages. George can also be consulted for additional testing vs. Quantum deliverables. 

Unit tests are passing (skipping Excel ones):
```
tests\test_banked_chains.py ....                                         [  1%]
tests\test_batch.py ......................                               [  9%]
tests\test_chain.py ...............                                      [ 15%]
tests\test_chain_old.py .....                                            [ 17%]
tests\test_cluster.py .......                                            [ 19%]
tests\test_complex_logic.py ..................................           [ 32%]
tests\test_dataset.py ........................................           [ 47%]
tests\test_io_dimensions.py .                                            [ 48%]
tests\test_link.py ....                                                  [ 49%]
tests\test_logic_views.py .                                              [ 50%]
tests\test_merging.py .......                                            [ 52%]
tests\test_recode.py .                                                   [ 53%]
tests\test_rim.py ...                                                    [ 54%]
tests\test_rules.py ...............                                      [ 59%]
tests\test_stack.py ................................                     [ 71%]
tests\test_view_manager.py .......................                       [ 80%]
tests\test_view_mapper.py ......                                         [ 82%]
tests\test_view_maps.py ....................................             [ 96%]
tests\test_weight_engine.py .......                                      [ 98%]
tests\test_xlsx_formats.py ...        
================= 266 passed, 5 deselected in 434.25 seconds ==================
```

### CODE REVIEW

**Comment by alexbuchhammer** [comment]
@AndersFreund @LiaStefan This can be reviewed now.

File: quantipy/sandbox/sandbox.py [hunk 1]
```diff
@@ -2811,9 +2811,13 @@ def get(self, data_key, filter_key, x_keys, y_keys, views, rules=False,
...
-
-            self._frame = pd.concat(self._pad(x_frames), axis=self.axis)
-
+            try:
+                self._frame = pd.concat(self._pad(x_frames), axis=self.axis)
+            except:
+                for f in x_frames:
+                    print f
+                    print
+                raise
```
Comment (AndersFreund) [inline quantipy/sandbox/sandbox.py]: You want this to stay here?

File: quantipy/core/quantify/engine.py [hunk 2]
```diff
@@ -2302,6 +2347,79 @@ def _apply_base_flags(self, sigres, replace=True):
...
+        diagnostics: tuple
+            The collection of all interim figures, i.e.:
+            * chi sq cell matrix
+            * cell counts
+            * sample and subsample proportions
+        """
+        valid_levels = {
+            0.01: 2.71, # 90%
+            0.05: 3.84, # 95%
+            0.1: 6.63 # 99%
```
Comment (AndersFreund) [inline quantipy/core/quantify/engine.py]: @alexbuchhammer I think the comments are off. Top one is 99% and bottom one is 90%

**Review by LiaStefan — APPROVED** [review]

File: quantipy/sandbox/sandbox.py (hunk 1 above)
Comment (alexbuchhammer) [reply quantipy/sandbox/sandbox.py]: Fixed!

File: quantipy/core/quantify/engine.py (hunk 2 above)
Comment (alexbuchhammer) [reply quantipy/core/quantify/engine.py]: You're right! Fixed.

**Review by AndersFreund — APPROVED** [review]

**Pull request merged**